target/linux/ath79/dts/ar7240_engenius_enh202-v1.dts

```diff
@@ -71,6 +71,24 @@
 	};
 };
 
+&partitions {
```

Comment:
I don't think the DTSI should be cut down further here. Since the DTSI only contains the partitioning anyway (and actually all devices using it are 4k), please just apply the changes to the DTSI. When we actually will have multiple partitioning schemes for ar724x at some point, we can revisit this and rename the DTSI to something-8m.dtsi eventually.

Reply:
the mutliple partitioning schemes is actually required now (to fix config restore)

ENH202 v1 is 64k
the other boards are 4k

I will go ahead and make the 2 different DTSIs now

Reply:
Err, just looked at it properly: Why is enh202-v1 not set to 4k, though it's tiny as well?
Is that what you mean by "therefore partitioning must be different for these devices depending on the flash chip model..."? Please make that clearer, also by actually naming the devices you speak about.
Even if that holds, I'd not split down the DTSI further, and create something-4k.dtsi and something-64k.dtsi instead.

Reply:
Yes exactly. It depends on the flash chip model. I will make this more clear in the commit message

for ENH202, the kernel detects the flash as "m25p64" which does not have SECT_4K symbol

`[    0.332017] spi-nor spi0.0: m25p64 (8192 Kbytes)`

https://github.com/torvalds/linux/blob/master/drivers/mtd/spi-nor/micron-st.c#L197

Reply:
ok done 👍🏼